virtualNetwork.tf

```diff
@@ -1,7 +1,33 @@
-locals {
-  vnet_address_cidr = tonumber(split("/", var.vnet_address_space)[1])
+module "subnet_addrs" {
+  source = "hashicorp/subnets/cidr"
+  version = "~> 1.0.0"
+
+  base_cidr_block = var.vnet_address_space
+  networks = [
+    {
+      name     = "AzureFirewallSubnet"
+      new_bits = 10
```

Comment:
What does this value represent? Is it the same meaning as the native function used before? If so, we need to make this dynamic because we don't know what the CIDR of the VNet space will be.